executor/executor_test.go

```diff
@@ -2204,6 +2204,49 @@ func (s *testSuite) TestAggregation(c *C) {
 	tk.MustExec("insert into t1 (a) values (1),(2),(3),(4),(5)")
 	result = tk.MustQuery("select count(a) from t1 where a < 3")
 	result.Check(testkit.Rows("2"))
+	tk.MustExec("drop table if exists t1")
```

Comment:
Please add comment for this. For example "The following cases use streamed aggregation."